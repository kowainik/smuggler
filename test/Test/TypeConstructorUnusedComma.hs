module Test.TypeConstructorUnusedComma where

import Data.Bool (Bool (False, True))

false = False
