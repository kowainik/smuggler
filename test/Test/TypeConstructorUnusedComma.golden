module Test.TypeConstructorUnusedComma where

import Data.Bool (Bool (False))

false = False
